<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>topping</class>
 <widget class="QWidget" name="topping">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>1380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>1380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>1161</width>
     <height>1401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/topping_page/topping_page_back.png);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>245</width>
     <height>360</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>245</width>
     <height>360</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/topping_page/click_off/t_1.png);</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>600</y>
     <width>245</width>
     <height>360</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>245</width>
     <height>360</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/topping_page/click_off/t_7.png);</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>600</y>
     <width>245</width>
     <height>360</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>245</width>
     <height>360</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/topping_page/click_off/t_8.png);</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>230</y>
     <width>245</width>
     <height>360</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>245</width>
     <height>360</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/topping_page/click_off/t_4.png);</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>600</y>
     <width>245</width>
     <height>360</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>245</width>
     <height>360</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/topping_page/click_off/t_6.png);</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>245</width>
     <height>360</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>245</width>
     <height>360</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/topping_page/click_off/t_5.png);</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>230</y>
     <width>245</width>
     <height>360</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>245</width>
     <height>360</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/topping_page/click_off/t_3.png);</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>245</width>
     <height>360</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>245</width>
     <height>360</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/topping_page/click_off/t_2.png);</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
